ENGAGEHF/Dashboard/Notifications/Views/NotificationRow.swift

```diff
@@ -0,0 +1,111 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+
+
+struct NotificationRow: View {
+    @Binding var notification: Notification
+    @State private var isExpanded = false
+    
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            HStack(alignment: .center) {
+                Text(notification.type)
+                    .font(.subheadline)
+                    .foregroundStyle(.secondary)
+                Spacer()
+                XButton(notification: $notification)
+            }
+            Divider()
+            Text(notification.title)
+                .multilineTextAlignment(.leading)
+                .padding(.bottom, 10)
```

Comment:
I would suggest that this text could be a bit smaller and could use a bold font to clearly highlight it as a title. The padding to the bottom could also be smaller.